source/Account.cpp

```diff
@@ -127,7 +127,7 @@ void Account::PayExtra(int mortgage, int64_t amount)
 
 
 // Step forward one day, and return a string summarizing payments made.
-string Account::Step(int64_t assets, int64_t salaries, int64_t maintenance)
+string Account::Step(int64_t assets, int64_t salaries, int64_t maintenance, int64_t escortCosts)
```

Comment:
```suggestion
string Account::Step(int64_t assets, int64_t salaries, int64_t maintenance, int64_t escortSalaries)
```
For coherence lets always use escortSalaries (you could also just use escortCosts everywhere if you prefer that but it needs to be coherent, I personally prefer salary because it reflects it is a daily cost even if you're not actually paying for their crew salaries, you're paying for the ship)